src/app/core/services/search-registration/search-registration.service.ts

```diff
@@ -27,6 +29,7 @@ export class SearchResult<TViewModel> {
   static DEBUG_TAG = 'SearchResult';
   readonly registrations$: Observable<TViewModel[]>;
   private forceUpdate = new Subject<void>();
+  isFetching$ = new BehaviorSubject<boolean>(false);
```

Comment:
Så vidt jeg vet er det vanlig bruke $ på slutten av variabelnavn kun for Observable og ikke Subject. 
Hvis ingen utenfor denne klassen har behov for å endre på isFetching$, tror jeg det er tryggere å eksponere denne som en Observable.